packages/wdio-runner/src/index.ts

```diff
@@ -75,11 +75,16 @@ export default class Runner extends EventEmitter {
         /**
          * add built-in services
          */
+        const softAssertionService = new SoftAssertionService({
+            autoAssertOnTestEnd: this._config.autoAssertOnTestEnd || true
```

Comment:
Shouldn't we add `autoAssertOnTestEnd` as configuration to the docs and `constants.ts`?